src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -713,6 +715,28 @@ public void deregisterWorker(String workerName) throws IOException {
     return workerSet;
   }
 
+  // When performing a graceful scale down of workers, the backplane can provide worker names to the
+  // scale-down service. The algorithm in which the backplane chooses these workers can be made more
+  // sophisticated in the future. But for now, we'll give back n random workers.
+  public List<String> SuggestedWorkersForShutdown(int amount) throws IOException {
```

Comment:
Should be suggestedWorkersForShutdown

Reply:
Also, maybe the method name should be something like suggestedWorkersToScaleDown and amount should be more descriptive, maybe numWorkers.

Reply:
both suggestions are better. fixed.